tests/test_pod.py

```diff
@@ -110,17 +117,16 @@ def test_pod_launcher_cleanup(
     assert "did not become healthy" in str(rt_err_ctx.value)
 
     # the pod should be gone
-    assert (
-        name
-        in host.run_expect([125], f"podman pod inspect {name}").stderr.strip()
-    )
+    with pytest.raises(
+        RuntimeError, match=f"Container with id {name} not found"
+    ):
+        container_runtime.inspect_container(name)
```

Comment:
This will inspect the container `i_will_fail_to_launch` not the **pod** `i_will_fail_to_launch`. As we never created that container, this will fail **always**

Reply:
Ah, missed the "pod" argument. Will fix this.